src/components/creator/Creator.jsx

```diff
@@ -23,7 +23,7 @@ export default function Creator(props) {
             onDelete={onDelete}
             options={options}
             hidden={!!item._destroy} // eslint-disable-line no-underscore-dangle
-            selected={item[selectedField]}
+            selected={item[selectedField].toString()}
```

Comment:
For reference, not checking `item[selectedField]` makes it blow up if we delete a BME. Thinking about all of this.